tests/system-tests/diskencryption/internal/file/file.go

```diff
@@ -0,0 +1,27 @@
+package file
```

Comment:
Could you please add these functions under `tests/system-tests/internal/files` ?

Reply:
fixed